packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.test.tsx

```diff
@@ -600,7 +601,7 @@ describe('SpinButton', () => {
 
   it(`should pass KeyCode ${KeyCodes.enter} to onValidate handler`, () => {
     let keyCode;
-    const onValidate: jest.Mock = jest.fn((value, event) => {
+    const onValidate: jest.Mock = jest.fn((value: any, event: any) => {
```

Comment:
Should these be any, or should we disable typedef for *.test files?

Reply:
I would be totally ok with disabling for *.test files.

Maybe even for *.example as well. The typings kind of pollute the message for the example.